auth/github/docs/api/readme.md

```diff
@@ -0,0 +1,278 @@
+---
+layout: api
+page_title: GitHub - Auth Methods - HTTP API
+description: This is the API documentation for the OpenBao GitHub auth method.
+---
+
+# GitHub auth method (API)
+
+This is the API documentation for the OpenBao GitHub auth method. For
+general information about the usage and operation of the GitHub method, please
+see the [OpenBao GitHub method documentation](/bao/docs/auth/github).
+
+This documentation assumes the GitHub method is enabled at the `/auth/github`
+path in OpenBao. Since it is possible to enable auth methods at any location,
+please update your API calls accordingly.
+
+## Configure method
+
+Configures the connection parameters for GitHub. This path honors the
+distinction between the `create` and `update` capabilities inside ACL policies.
+
+| Method | Path                  |
+| :----- | :-------------------- |
+| `POST` | `/auth/github/config` |
+
+### Parameters
+
+- `organization` `(string: <required>)` - The organization users must be part
+  of.
+- `organization_id` `(int: 0)` - The ID of the organization users must be part
+  of. OpenBao will attempt to fetch and set this value if it is not provided.
+- `base_url` `(string: "")` - The API endpoint to use. Useful if you are running
+  GitHub Enterprise or an API-compatible authentication server.
+
+### Environment variables
+- `VAULT_AUTH_CONFIG_GITHUB_TOKEN` `(string: "")` - An optional GitHub token used to make
+  authenticated GitHub API requests. This can be useful for bypassing GitHub's
+  rate-limiting during automation flows when the `organization_id` is not provided.
+  We encourage you to provide the `organization_id` instead of relying on this environment variable.
```

Comment:
```suggestion
```

This is also no longer supported as a standalone plugin

Reply:
is updated